pkg/runtime/runtime-manifest.go

```diff
@@ -196,7 +195,7 @@ func (m *RuntimeManifest) setStepsByAction() error {
 			for a := range m.CustomActions {
 				actions = append(actions, a)
 			}
-			errors.Errorf("unsupported action %q, custom actions are defined for: %s", m.Action, strings.Join(actions, ", "))
+			// fmt.Errorf("unsupported action %q, custom actions are defined for: %s", m.Action, strings.Join(actions, ", "))
```

Comment:
⚠️ **FIXME**: remove comment (🤯 misclick keyboard shortcut probably)

Reply:
This should return the error.